proposal/color-4-hwb.md

```diff
@@ -82,22 +105,37 @@ Sass named-argument syntax.
   * If any of `$hue`, `$whiteness`, `$blackness`, or `$alpha` aren't numbers,
     throw an error.
 
-  * If either of `$whiteness` or `$blackness` is not a percentage between 0% and
-    100%, throw an error.
+  * If `$hue` has any units other than `deg`, throw an error.
```

Comment:
CSS accepts several different `<angle>` units. I don't know how much they are used (`deg` is certainly the most common), but we might want to match? https://www.w3.org/TR/css3-values/#angle-value

Reply:
We should support that, but it's tangential enough that I don't think it should be part of this proposal. I've filed https://github.com/sass/sass/issues/2904 to track that.